### ISSUE TITLE
Webviews don't take screen reader focus on opening

### ISSUE THREAD

**Issue by jackjlynch** [microsoft/vscode#94229]
- VSCode Version: 1.4.3
- OS Version: Windows 10

Steps to Reproduce:

1. With NVDA or Narrator running, open a webview panel (e.g. Release Notes)
2. Focus moves inside the panel, but the screen reader focus does not, instead just reading "Virtual Document document." Keyboard navigation and tabbing within the webview work, but the screen reader will not read anything unless you click within the webview using the mouse.

Does this issue occur when all extensions are disabled?: Yes. Initially discovered with [Elements for VS Code](https://github.com/microsoft/vscode-edge-devtools) but reproes with other webviews as well

Maybe related to #84042

**Comment by isidorn** [comment]
@jackjlynch thanks for filling this issue. We have changed the role of webviews to be `document` with latest vscode insiders. Can you also repro this issue with latest vscode insiders?
I just quickly tried it with VoiceOver and VoiceOver first reads that the Document is focused and that a key combination should be pressed to focus into it.

fyi @marcozehe

**Comment by MarcoZehe** [comment]
Yes I believe this is a duplicate of my issue, and once NVDA 2020.1 stops turning off browse mode again, something which just recently got introduced, this should just start working as expected.

Note that MacOS doesn't have a concept of a virtual cursor like NVDA and JAWS do, but still recognizes that certain areas are documents in Insider builds now, and gives appropriate prompts.

**Comment by jackjlynch** [comment]
@isidorn It doesn't repro for me on latest insider. NVDA still reads out "Virtual Document document" initially but seems to follow keyboard focus after that. Thanks

**Comment by isidorn** [comment]
Thanks for letting us know @jackjlynch
